Source/Charts/Utils/ChartColorTemplates.swift

```diff
@@ -84,6 +84,18 @@ open class ChartColorTemplates: NSObject
         ]
     }
     
+    @objc open class func gradients () -> [[NSUIColor]]
+    {
+        return [
+            [NSUIColor(red: 146/255.0, green: 132/255.0, blue: 240/255.0, alpha: 1.0),
```

Comment:
Where did these colours come from? If they are from other templates, you should directly call them instead of redefining them.

Reply:
It's custom colours especially for the gradients.

Reply:
Right, but where did the values for the colours come from? We already have a list of themes in `ChartColorTemplates`. If this adds to that, it should be done there. It probably should just use the already existing theme colours though.